typescript/forms.d.ts

```diff
@@ -1183,41 +1183,46 @@ export type FormElementLookupStaticDataRecordBase = {
   preFills: FormElementLookupStaticDataPreFill[]
 }
 
-export type FormElementLookupStaticDataRecordText = FormElementLookupStaticDataRecordBase & {
-  inputType: 'TEXT'
-  /**
-   * The value that will be matched exactly on the form element this lookup is
-   * associated when the user is completing the form.
-   */
-  inputValue: string
-}
-export type FormElementLookupStaticDataRecordNumber = FormElementLookupStaticDataRecordBase & {
-  inputType: 'NUMBER'
-  /**
-   * The value that will be matched exactly on the form element this lookup is
-   * associated when the user is completing the form.
-   */
-  inputValue: number
-}
+export type FormElementLookupStaticDataRecordText =
```

Comment:
Think we want a new type of record, e.g.
```
export type FormElementLookupStaticDataRecordUndefined =
  FormElementLookupStaticDataRecordBase & {
    inputType: 'UNDEFINED'
  }
```
But I havent thought about the implications of the above config in the renderer